src/styles/blocks/review.scss

```diff
@@ -0,0 +1,16 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 16px;
+
+  &__num {
+    display: flex;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: #060b35;
```

Comment:
Checklist 10 suggests using variables. Maybe you should convert some repeating values into them.

Reply:
Done